lib/hooks/api/useContributorList.ts

```diff
@@ -30,16 +30,15 @@ export const useContributorsList = ({
     meta: Meta;
   };
   initialPage?: number;
-  defaultLimit?: number;
+  defaultLimit?: string;
   defaultRange?: string;
 }) => {
-  const [range, setRange] = useState(defaultRange); // [start, end
   const [page, setPage] = useState(initialPage);
-  const [limit, setLimit] = useState(defaultLimit);
+
   const query = new URLSearchParams();
   query.append("page", page.toString());
-  query.append("limit", limit.toString());
-  query.append("range", range);
+  query.append("limit", defaultLimit ?? "10");
```

Comment:
Refactor (blocking): The `defaultLimit` already has a default value of `"10"`, so `?? "10"` is note required.

```suggestion
  query.append("limit", defaultLimit);
```